src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalAnalysisResult.js

```diff
@@ -284,80 +290,69 @@ const ClinicalAnalysisResult = ({
             setId={setId}
             />
 
-          <Column
+          <ul
+            className="masonry"
             style={{
-              flex: 4,
+              padding: '0 0 0 0.7%',
               minWidth: 0,
             }}
             >
-            <Column
-              className="print-grid"
-              style={{
-                display: 'grid',
-                gridTemplateColumns: controlPanelExpanded
-                  ? '50% 50%'
-                  : '33% 33% 33%',
-                gridTemplateRows: 'repeat(auto)',
-                ...(controlPanelExpanded ? {} : { marginLeft: '1%' }),
-              }}
+            <li
+              className="masonry-brick"
```

Comment:
this list item needs a `key`

```suggestion
              className="masonry-brick"
              key="clinicalanalysis-overallsurvival"
```